### ISSUE TITLE
Can we remove EclipseInterface from TestNG?

### ISSUE THREAD

**Issue by juherr** [cbeust/testng-eclipse#318]
From: https://github.com/cbeust/testng/blob/master/src/main/java/org/testng/internal/EclipseInterface.java#L4-L5

> Symbols in this class are used by the Eclipse plug-in, do not modify them without updating the plug-in as well.

But I'm not able to find any reference here. Do you still use it or can we remove the class from TestNG?

**Comment by missedone** [comment]
looks like we can remove this (or rename it since `org.testng.Assert` references it).
the only thing related in Eclipse might be in `CompareResultDialog` to extract diff with regex:
```java
static String[] regexes = { 
      ".*expected:<(.*)> but was:<(.*)>.*", 
      ".*expected not same with:<(.*)> but was same:<(.*)>.*", 
      ".*expected same with:<(.*)> but was:<(.*)>.*", 
      ".*expected \\[(.*)\\] but found \\[(.*)\\].*" 
      };
```

@cbeust , could you confirm?

**Comment by cbeust** [comment]
Yes, I think they were only there for `CompareResultDialog`. Make sure this still works without them.

**Comment by missedone** [comment]
@juherr, you can remove or rename it as long as keep the assertion message patterns, so that won't break eclipse plugin.
